.devcontainer/devcontainer.json

```diff
@@ -79,7 +79,7 @@
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
 	"onCreateCommand": "cd /workspaces/composable/scripts/polkadot-launch && yarn && cd /workspaces/composable/integration-tests/runtime-tests && npm install",
-	"postCreateCommand": "mkdir -p /workspaces/polkadot/target/release && cp /polkadot-binary/polkadot /workspaces/polkadot/target/release/polkadot",
+  	"postCreateCommand": "mkdir -p /workspaces/polkadot/target/release && cp /polkadot-binary/polkadot /workspaces/polkadot/target/release/polkadot && mkdir -p /workspaces/Basilisk-node/target/release && cp /Basilisk-node/basilisk /workspaces/Basilisk-node/target/release/basilisk",
```

Comment:
spaces?

Reply:
Don't know why, but for some reason after the first commit the line lost its alignment.
Tabbed it in once in the second commit. Don't know why it thinks it's different though.